app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -6,7 +6,6 @@ CONFIG_INTEL_ALH=n
 CONFIG_RIMAGE_SIGNING_SCHEMA="mtl"
 CONFIG_IPC_MAJOR_4=y
 CONFIG_INTEL_ADSP_IPC=y
-CONFIG_DW_ICTL_ACE_V1X=y
```

Comment:
Is was renamed to CONFIG_DW_ICTL_ACE. Please change option name instead of delete it

Reply:
It should be removed as it is already selected for ACE

Reply:
Ack. @softwarecki  I confirmed with a local mtl build that CONFIG_DW_ICTL_ACE is defined in the build even if we leave it out here on SOF side. I'll anyways update the PR to include the dmic fix.